.github/workflows/dotnet.yml

```diff
@@ -9,12 +9,12 @@ on:
 jobs:
   build:
     environment: prod
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
I don't get the point of this. If it's not broken, don't change it.
Linux is not a better OS to test .NET than macOS.

Reply:
The root cause is GitHub drops support for .NET Core 3.1. I updated it to .NET 8.0 and now it works.

Reply:
Back merged master in the PR.